packages/vue/src/composables/table.ts

```diff
@@ -161,6 +161,10 @@ export function useTable<
 		selectAll: bulk.selectAll,
 		/** Deselects all records. */
 		deselectAll: bulk.deselectAll,
+		/** Selects paged records. */
+		selectPage: bulk.select(...table.value.records.map((record: RecordType) => getRecordKey(record))),
+		/** Deselects paged records. */
+		deselectPage: bulk.deselect(...table.value.records.map((record: RecordType) => getRecordKey(record))),
```

Comment:
This seems wrong. `selectPage` and `deselectPage` are not functions, since the underlying `select` and `deselect` are called at the moment they are defined. 

```ts
const { selectPage } = useTable()

// `selectPage` is not a function. It's just `void`.
```